go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/corpix/uarand v0.1.1 // indirect
 	github.com/cyberdelia/go-metrics-graphite v0.0.0-20161219230853-39f87cc3b432
 	github.com/evanphx/json-patch v4.5.0+incompatible
+	github.com/falun/watch v0.0.0-20200621174039-43c45ed8922d
```

Comment:
Nothing against your lib ! but what about `github.com/fsnotify/fsnotify` ?

Reply:
> Nothing against your lib ! but what about `github.com/fsnotify/fsnotify` ?

Mostly getting it to work with my thing was essentially trivial to drop in and not the interesting part of the change. I'll swap out for a tested solution before actually baking the review

Reply:
oh ugh. I remember thinking about this now

So the fancy way of doing this with notifications is that it basically is hard to operate on a file that gets created/renamed/deleted/doesn't exist. The dumb-as-rocks version i used in my library is _totally fine_ with that since it's just polling.

I mostly don't have strong feelings about this and don't want to fight upstream about using my jank so I'm still going to try swapping out for fsnotify but that's the underlying reason why it might be preferred to do the simple thing instead.